velox/exec/MergeJoin.h

```diff
@@ -384,7 +391,7 @@ class MergeJoin : public Operator {
     bool currentRowPassed_{false};
   };
 
-  std::optional<LeftJoinTracker> leftJoinTracker_{std::nullopt};
+  std::optional<JoinTracker> joinTracker_{std::nullopt};
```

Comment:
Could we add some comments for this variable to illustrate it is for left and right join?